src/Models/Classes/SceneView.types.ts

```diff
@@ -107,6 +131,8 @@ export interface ISceneViewProps {
     cameraPosition?: ICameraPosition;
     coloredMeshItems?: CustomMeshItem[];
     getToken?: () => Promise<string>;
+    gizmoElementItem?: TransformedElementItem;
```

Comment:
if one of these gizmo props is provided, are all of them required? or are each independent?